Kokoro Windows tests fail on CloudSdk.validateAppEngineJavaComponents()

(Note that the error message `Could not delete project` below is just a mis-typed assertion message! It should be `Could not create project`. To be fixed in https://github.com/GoogleCloudPlatform/google-cloud-eclipse/pull/1901.)

Every call to `CloudSdk.validateAppEngineJavaComponents()` fails starting today, and my guess is that Kokoro started to pick up an old gcloud version pre-configured on the Kokoro Windows env: https://github.com/GoogleCloudPlatform/appengine-plugins-core/pull/397#issuecomment-298937953.

```
java.lang.AssertionError: Could not delete project
	at com.google.cloud.tools.appengine.cloudsdk.CloudSdk.validateAppEngineJavaComponents(CloudSdk.java:507)
	at com.google.cloud.tools.eclipse.appengine.facets.AppEngineFacetInstallDelegate.validateAppEngineJavaComponents(AppEngineFacetInstallDelegate.java:44)
	at com.google.cloud.tools.eclipse.appengine.facets.AppEngineFacetInstallDelegate.execute(AppEngineFacetInstallDelegate.java:36)
	at com.google.cloud.tools.eclipse.appengine.facets.StandardFacetInstallDelegate.execute(StandardFacetInstallDelegate.java:52)
	at org.eclipse.wst.common.project.facet.core.internal.FacetedProject.callDelegate(FacetedProject.java:1477)
	at org.eclipse.wst.common.project.facet.core.internal.FacetedProject.modifyInternal(FacetedProject.java:441)
	at org.eclipse.wst.common.project.facet.core.internal.FacetedProject.mergeChangesInternal(FacetedProject.java:1181)
	at org.eclipse.wst.common.project.facet.core.internal.FacetedProject.access$2(FacetedProject.java:1117)
	at org.eclipse.wst.common.project.facet.core.internal.FacetedProject$1.run(FacetedProject.java:324)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2240)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2267)
	at org.eclipse.wst.common.project.facet.core.internal.FacetedProject.modify(FacetedProject.java:339)
	at com.google.cloud.tools.eclipse.appengine.facets.FacetUtil.install(FacetUtil.java:200)
	at com.google.cloud.tools.eclipse.test.util.project.TestProjectCreator.addFacets(TestProjectCreator.java:164)
	at com.google.cloud.tools.eclipse.test.util.project.TestProjectCreator.createProject(TestProjectCreator.java:138)
	at com.google.cloud.tools.eclipse.test.util.project.TestProjectCreator.createProjectIfNecessary(TestProjectCreator.java:122)
	at com.google.cloud.tools.eclipse.test.util.project.TestProjectCreator.getProject(TestProjectCreator.java:109)
	at com.google.cloud.tools.eclipse.appengine.deploy.ui.standard.DeployPropertyPageForStandardProjectTest.getProject(DeployPropertyPageForStandardProjectTest.java:37)
```

--- a/kokoro/windows/continuous.bat
+++ b/kokoro/windows/continuous.bat
@@ -4,7 +4,7 @@ set CLOUDSDK_CORE_DISABLE_USAGE_REPORTING=true
 call gcloud.cmd components update --quiet
 call gcloud.cmd components install app-engine-java --quiet
 
-mvn --version
-mvn -B --fail-at-end -Ptravis -Declipse.target=oxygen verify
+mvn -B -N io.takari:maven:wrapper -Dmaven=3.5.0
+mvnw.cmd -B --fail-at-end -Ptravis -Declipse.target=oxygen verify
 
 exit /b %ERRORLEVEL%

--- a/kokoro/windows/continuous.bat
+++ b/kokoro/windows/continuous.bat
@@ -1,16 +1,16 @@
 cd github/google-cloud-eclipse
 
 set CLOUDSDK_CORE_DISABLE_USAGE_REPORTING=true
-wget https://dl.google.com/dl/cloudsdk/channels/rapid/GoogleCloudSDKInstaller.exe
-start /WAIT GoogleCloudSDKInstaller.exe /S /noreporting /nostartmenu /nodesktop /logtofile /D=T:\google
-call t:\google\google-cloud-sdk\bin\gcloud.cmd components copy-bundled-python>>python_path.txt && SET /p CLOUDSDK_PYTHON=<python_path.txt && DEL python_path.txt
+wget -nv https://dl.google.com/dl/cloudsdk/channels/rapid/GoogleCloudSDKInstaller.exe
+start /wait GoogleCloudSDKInstaller.exe /S /noreporting /nostartmenu /nodesktop /logtofile /D=T:\google
+call t:\google\google-cloud-sdk\bin\gcloud.cmd components copy-bundled-python>>python_path.txt && set /p CLOUDSDK_PYTHON=<python_path.txt && del python_path.txt
 call t:\google\google-cloud-sdk\bin\gcloud.cmd components update --quiet
 call t:\google\google-cloud-sdk\bin\gcloud.cmd components install app-engine-java --quiet
 set GOOGLE_CLOUD_SDK_HOME=t:\google\google-cloud-sdk
 
-wget http://www-us.apache.org/dist/maven/maven-3/3.5.0/binaries/apache-maven-3.5.0-bin.zip
-unzip apache-maven-3.5.0-bin.zip
+wget -nv http://www-us.apache.org/dist/maven/maven-3/3.5.0/binaries/apache-maven-3.5.0-bin.zip
+unzip -q apache-maven-3.5.0-bin.zip
 
-apache-maven-3.5.0/bin/mvn --fail-at-end -Ptravis -Declipse.target=oxygen verify
+apache-maven-3.5.0/bin/mvn -B --fail-at-end -Ptravis -Declipse.target=oxygen verify
 
 exit /b %ERRORLEVEL%

--- a/kokoro/windows/continuous.bat
+++ b/kokoro/windows/continuous.bat
@@ -1,16 +1,10 @@
 cd github/google-cloud-eclipse
 
 set CLOUDSDK_CORE_DISABLE_USAGE_REPORTING=true
-wget -nv https://dl.google.com/dl/cloudsdk/channels/rapid/GoogleCloudSDKInstaller.exe
-start /wait GoogleCloudSDKInstaller.exe /S /noreporting /nostartmenu /nodesktop /logtofile /D=T:\google
-call t:\google\google-cloud-sdk\bin\gcloud.cmd components copy-bundled-python>>python_path.txt && set /p CLOUDSDK_PYTHON=<python_path.txt && del python_path.txt
-call t:\google\google-cloud-sdk\bin\gcloud.cmd components update --quiet
-call t:\google\google-cloud-sdk\bin\gcloud.cmd components install app-engine-java --quiet
-set GOOGLE_CLOUD_SDK_HOME=t:\google\google-cloud-sdk
+call gcloud.cmd components update --quiet
+call gcloud.cmd components install app-engine-java --quiet
 
-wget -nv http://www-us.apache.org/dist/maven/maven-3/3.5.0/binaries/apache-maven-3.5.0-bin.zip
-unzip -q apache-maven-3.5.0-bin.zip
-
-apache-maven-3.5.0/bin/mvn -B --fail-at-end -Ptravis -Declipse.target=oxygen verify
+mvn --version
+mvn -B --fail-at-end -Ptravis -Declipse.target=oxygen verify
 
 exit /b %ERRORLEVEL%

